.release-it.json

```diff
@@ -0,0 +1,17 @@
+{
+  "git": {
+    "requireBranch": "master",
+    "commitMessage": "Version ${version}",
+    "tagName": "${version}",
+    "tagAnnotation": "Version ${version}"
+  },
+  "github": {
+    "release": true,
+    "releaseName": "Version ${version}",
+    "draft": true,
+    "tokenRef": "GITHUB_TOKEN"
```

Comment:
Where is this coming from? Should the dev doing the release have this configured?

Reply:
Yes, this is an environmental variable that needs to be set in the CI and on the dev machine.
I'm leaving here a [link](https://github.com/release-it/release-it/blob/master/docs/environment-variables.md) to the doc for reference.